crates/biome_js_analyze/src/lint/nursery/no_enum.rs

```diff
@@ -1,6 +1,6 @@
 use biome_analyze::{context::RuleContext, declare_lint_rule, Ast, Rule, RuleDiagnostic};
 use biome_console::markup;
-use biome_js_syntax::TsEnumDeclaration;
+use biome_js_syntax::{JsFileSource, TsEnumDeclaration};
 use biome_rowan::AstNode;
 
 declare_lint_rule! {
```

Comment:
In the docs, we could say that `const enum` are not covered by this rule, because `noCOnstEnum` already rule out them. We could also say that enum in ambient context (including in declaration files) are ignored.

Reply:
I have updated the docs, feel free to review them if you think I could add more info! thanks!